.storybook/preview-head.html

```diff
@@ -0,0 +1,4 @@
+<link
```

Comment:
If we are doing this, it means this has be in the docs in the README as a requirement to use the components.